api-tester/src/defaults/kotlin/com/revenuecat/apitester/kotlin/PurchasesAPI.kt

```diff
@@ -146,4 +152,11 @@ private class PurchasesAPI {
 
         Purchases.debugLogsEnabled = true
     }
+
+    fun checkLogInResult(
+        logInResult: LogInResult,
```

Comment:
Since we are also exposing the constructor for `LogInResult`, we should also add the constructor to the API tests... Alternatively, we can make the constructor internal. Users shouldn't use it, but might be useful for them to test...